packages/react-storage/src/components/StorageBrowser/controls/types.ts

```diff
@@ -43,5 +43,10 @@ export interface ControlsContext {
       | 'LIST_LOCATIONS'
       | 'LIST_LOCATION_ITEMS';
     isCancelable?: boolean;
+    actionStart?: {
+      onClick?: () => void;
+      label?: string;
+      isDisabled?: boolean;
+    };
```

Comment:
This isn't set in stone by any means yet but I think the final shape of this this will be closer to something like:
```ts
export interface ControlsContext {
  data: {
    taskCounts?: TaskCounts;
    tableData?: TableData;
    actionStartLabel?: string;
    isActionStartDisabled?: boolean;
  };
  actionsConfig: {
    type:
      | 'SINGLE_ACTION'
      | 'BATCH_ACTION'
      | 'LIST_LOCATIONS'
      | 'LIST_LOCATION_ITEMS';
    isCancelable?: boolean;
  },
  onActionStart?: () => void;
}
```
I.e. .. please use this for now 🙏 😅